specification/autoscaling/put_autoscaling_policy/autoscalingApisPutAutoscalingPolicyRequestExample2.json

```diff
@@ -0,0 +1,7 @@
+{
+  "summary": "Creates an autoscaling policy named `my_autoscaling_policy` using the fixed autoscaling decider, applying to the set of nodes having (only) the `data_hot` role",
```

Comment:
>  "summary": "Creates an autoscaling policy named `my_autoscaling_policy` using the fixed autoscaling decider, applying to the set of nodes having (only) the `data_hot` role",

... not a blocker but I think the summary ought to be shorter and this longer detail could move to the description.